test/unit/util/FetchUtil.test.ts

```diff
@@ -1,52 +1,69 @@
+import type { Quad } from '@rdfjs/types';
 import arrayifyStream from 'arrayify-stream';
 import { fetch } from 'cross-fetch';
 import { DataFactory } from 'n3';
+import rdfDereferencer from 'rdf-dereference';
 import { RdfToQuadConverter } from '../../../src/storage/conversion/RdfToQuadConverter';
-import { fetchDataset } from '../../../src/util/FetchUtil';
+import { fetchDataset, responseToDataset } from '../../../src/util/FetchUtil';
 const { namedNode, quad } = DataFactory;
 
 jest.mock('cross-fetch');
+jest.mock('rdf-dereference');
 
 describe('FetchUtil', (): void => {
+  const url = 'http://test.com/foo';
+
   describe('#fetchDataset', (): void => {
-    const fetchMock: jest.Mock = fetch as any;
-    const url = 'http://test.com/foo';
-    const converter = new RdfToQuadConverter();
+    const rdfDereferenceMock: jest.Mocked<typeof rdfDereferencer> = rdfDereferencer as any;
 
-    function mockFetch(body: string, status = 200): void {
-      fetchMock.mockImplementation((input: string): any => ({
-        text: (): any => body,
-        url: input,
-        status,
-        headers: { get: (): any => 'text/turtle' },
+    function mockDereference(quads?: Quad[]): any {
+      rdfDereferenceMock.dereference.mockImplementation((): any => ({
+        url,
+        quads,
+        exists: true,
       }));
```

Comment:
```suggestion
    function mockDereference(quads: Quad[]): any {
      (rdfDereferencer.dereference as jest.Mock).mockImplementation((): any => ({
        url,
        quads: Readable.from(quads),
        exists: true,
      }));
    }
```

Reply:
Yes, that is an oversight, of course I needed to wrap it in a Readable.from too..